### PR TITLE
relval matrix updates

### PR DESCRIPTION
**Pull request by hengne**
This is a collection of important matrix updates to go into the 8_0_1 production release:
- update golden json to the EOY re-reco complete version
- turn on multiCore with 4 cores for DIGI/HLT/RECO steps. N cores setting defined in workflow steps, and wmcontral read these numbers for relval production.
- re-arrange upgrade workflows removing old 8TeV workflows. 
- phase1 workflows set to 13TeV
- Update relval_2017 IB tests workflow list to be: [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0] 
- turn on realistic Beamspot for phase1 gen-sim, turn on phase1 GT for realistic BS, build on top of PR https://github.com/cms-sw/cmssw/pull/13394
- update phase1 gen-sim input 
- Add 3 data 2015 HighLumi run High stats workflows needed for cross POG studies
- Add a new TTbar workflow wrapped from an ext-gen workflow, starting from LHE generation (then pythia gen-sim, digi, reco, etc.,).
- Update minibias inputs for mixing and premixing.

### CODE REVIEW

**Comment by hengne** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -186,6 +186,13 @@
 steps['RunHLTPhy2015DHS']={'INPUT':InputInfo(dataSet='/HLTPhysics/Run2015D-v1/RAW',label='hltPhy2015DHS',events=100000,location='STD', ls=Run2015DHS)}
  
 
+#### run2 2015 HighLumi High Stat workflows ##
+# Run2015HLHS, 25ns, run 260627, JetHT: 2.9M, SingleMuon: 5.7M, ZeroBias: 1.6M
+Run2015HLHS=selectedLS([260627])
+steps['RunJetHT2015HLHS']={'INPUT':InputInfo(dataSet='/JetHT/Run2015D-v1/RAW',label='jetHT2015HLHT',events=100000,location='STD', ls=Run2015HLHS)}
```
Comment (davidlt) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: Is this part of standard IB workflows? If yes, the input block is not at T2_CH_CERN.

**Comment by davidlt** [comment]
Two things:
- Does this change the input files for CMSSW __IB RelVals__? If it does, have you checked that all required blocks are part of IB RelVals or RelVals PhEDEx and at T2_CH_CERN?
- Does this mean that we by default run some workflows in multi-threaded mode in CMSSW __IB RelVals__?

I am just trying to ensure we don't break IB RelVals because of missing files, do not get high IO impact because our input files in a different continent and we don't kill machines by overcommitting them with more threads.

File: Configuration/PyReleaseValidation/python/relval_steps.py (hunk 1 above)
Comment (hengne) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: no, these are for some on-demand requests. not to be tested in the IB test.

**Comment by hengne** [comment]
@davidlt 
Answers:

Q.1:  for the highstats relvals, they are not for IB tests, so nothing to worry. For phase1 workflows, the gen-sim inputs are produced and properly set.

Q.2:  Yes, the digi/hlt/reco steps in fullSim noPU/PU and data 2015 workflows are in 4 cores.

**Comment by hengne** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_2017.py [hunk 2]
```diff
@@ -15,7 +15,8 @@
 
 numWFStart=10000
 numWFSkip=200
-numWFIB = [10060.0,10039.0,10046.0] ##2017 WFs to run in IB (TenMuE_0_200, TTbar, MinBias)
+#2017 WFs to run in IB (TenMuE_0_200, TTbar, ZEE, MinBias, TTbar PU, ZEE PU)
+numWFIB = [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0]
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_2017.py]: Do these somewhat match the pattern of numbering in 2015?
It looks like ttbar doesn't.
Not that it matters much, but still.

Comment (hengne) [inline Configuration/PyReleaseValidation/python/relval_2017.py]: the numbering here do not match those workflows for 2015. It is simply a new mechanism of "auto-numbering" according to the order in a list, a mechanism newly introduced by Gaelle for the upgrade relvals.

**Comment by mmusich** [comment]
@hengne can you please include also this commit: https://github.com/cms-sw/cmssw/pull/13496?
It resumes testing of some AlCaRecos run at Express that got lost in some previous iteration.

**Comment by hengne** [comment]
@mmusich updated.

**Comment by hengne** [comment]
please test

**Comment by mmusich** [comment]
@hengne thanks!

**Comment by mmusich** [comment]
+1

**Comment by hengne** [comment]
+1

**Comment by hengne** [comment]
@davidlange6 

Hi David, here are two PRs to be merged before building the first 8_0_1 production release. 
- PR https://github.com/cms-sw/cmssw/pull/13394 from Marco for the GTs 
- this PR from me for Relval updates.

They might be not necessary for 800_patchX releases, but just to make sure to merge them before the production 8_0_1 release. Thanks! 

best, Hengne

**Comment by davidlange6** [comment]
@hengne - I need an 81x PR corresponding to this. I think it is not made yet

**Comment by hengne** [comment]
@davidlange6 ok, i am going to make this 81x pr right now.

**Comment by hengne** [comment]
@davidlange6 
done, here it is: https://github.com/cms-sw/cmssw/pull/13524

**Comment by hengne** [comment]
@davidlange6 I think i found the trick to use -t option in runTheMatrix but keep only the HARVEST step without impacted by it in both wmcontral and local run. And, all hardcoded --nThreads: 4 options in relval_steps.py are removed, so default in IB test will be 1 core.

**Comment by hengne** [comment]
please test

**Comment by fabozzi** [comment]
please test

**Comment by fabozzi** [comment]
+1

**Comment by mmusich** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**